### DISCUSSION TITLE
New cards without losing progress?

### DISCUSSION THREAD

**Discussion by tjthejuggler** [axelboc/anki-ultimate-geography#523] [Q&A]
Hello. I heard that you have a way for the authors of your deck to add new cards that users can add to their version of the deck without the user having to reset their deck history, is this true? If so, how do you do it?

**Comment by aplaice** [comment]
Do you mean adding new notes to an existing deck or adding new cards (by changing the note type) to existing notes in an existing deck?

If the former, then (as far as I recall) Anki already offers that capability.  If the author shares the deck as an an "apkg" (it's one of the Anki export options), then when the user imports an update of the deck, their history of existing cards will be kept.

If the author shares the deck as a CSV, then I think that the first fields have to be unique and unchanged (for existing notes), and the user has to check the "Update existing notes when first field matches" option.

If you want the latter (in addition to the former), then Anki doesn't easily allow it (AFAICT), but CrowdAnki (optionally with BrainBrew (which is what we use)) does. It's slightly more complicated, though, so if the former is enough for you, it might be overkill.
